Gemfile.lock

```diff
@@ -148,6 +148,12 @@ GEM
       unicode-display_width (~> 1.0, >= 1.0.1)
     ruby-progressbar (1.10.0)
     ruby_dep (1.5.0)
+    shoulda (3.6.0)
```

Comment:
This and some of the following lines, get removed when I run `bundle install` on this branch. It looks like you had these gems in your `Gemfile` and committed the resulting `Gemfile.lock`

Please re-run `bundle install` and commit the result.